### DISCUSSION TITLE
preview of post on page

### DISCUSSION THREAD

**Discussion by bendkaye613** [daattali/beautiful-jekyll#822] [Q&A]
I would like to show a preview of a post on a page, similar to what is done on the feed.

Is there a way to do this?

I can link to the posts on the page, but that's about it

**Comment by daattali** [comment]
This is not a supported feature. You'd have to learn how to use the jekyll template language and implement it yourself unfortunately

**Reply by bendkaye613** [reply]
I figured it out! I can do it if I tag the post and copy home.html

_this is what I did:_
1) copy home.html from _layouts and insert into my_page.html
2) set tags on desired posts
3) changed the following : 
`{% assign posts = paginator.posts | default: site.posts %}`
     to
 `{% assign posts = site.tags.TAG | default: site.tags.TAG %}`

I hope someone finds this useful!
